Tests/KernelTests/AccessValidator/AccessValidatorTests.cs

```diff
@@ -0,0 +1,220 @@
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+using US = LT.DigitalOffice.UserService.Broker.Consumers;
+using CRS = LT.DigitalOffice.CheckRightsService.Broker.Consumers;
+using LT.DigitalOffice.UserService.Repositories.Interfaces;
+using LT.DigitalOffice.CheckRightsService.Repositories.Interfaces;
+using AV = LT.DigitalOffice.Kernel.AccessValidator;
+using LT.DigitalOffice.UserService.Database.Entities;
+using System.Net;
+using Microsoft.Extensions.Primitives;
+
+namespace KernelUnitTests.AccessValidator
+{
+    public class AccessValidatorTests
+    {
+        private DbUser adminUser;
+        private DbUser notAdminUser;
+        private string userId;
+        private IAccessValidator accessValidator;
+
+        private const string USER_SERVICE_TEST_QUEUE = "UserService";
+        private const string USER_SERVICE_TEST_REQUEST_URL = "loopback://localhost/UserService";
+        private const string CHECK_RIGHTS_SERVICE_TEST_QUEUE = "CheckRightsService";
+        private const string CHECK_RIGHTS_SERVICE_TEST_REQUEST_URL = "loopback://localhost/CheckRightsService";
+        private const int RIGHT_ID = 5;
+
+        private InMemoryTestHarness harness;
+
+        private Mock<IHttpContextAccessor> httpContextMock;
+        private Mock<IOptions<RabbitMQOptions>> optionsMock;
+        private Mock<IUserRepository> userServiceRepositoryMock;
+        private Mock<ICheckRightsRepository> checkRightsServiceRepositoryMock;
+
+        [OneTimeSetUp]
+        public async Task OneTimeSetUp()
+        {
+            adminUser = new DbUser
+            {
+                IsAdmin = true
+            };
+            notAdminUser = new DbUser
+            {
+                IsAdmin = false
+            };
+            userId = Guid.NewGuid().ToString();
+
+            userServiceRepositoryMock = new Mock<IUserRepository>();
+            checkRightsServiceRepositoryMock = new Mock<ICheckRightsRepository>();
+
+            harness = new InMemoryTestHarness();
+
+            harness.Consumer(
+                () => new US.AccessValidatorConsumer(
+                    userServiceRepositoryMock.Object),
+                    USER_SERVICE_TEST_QUEUE);
+            harness.Consumer(
+                () => new CRS.AccessValidatorConsumer(
+                    checkRightsServiceRepositoryMock.Object),
+                    CHECK_RIGHTS_SERVICE_TEST_QUEUE);
+
+            httpContextMock = new Mock<IHttpContextAccessor>();
+
+            optionsMock = new Mock<IOptions<RabbitMQOptions>>();
+            optionsMock
+                .Setup(r => r.Value)
+                .Returns(new RabbitMQOptions
+                {
+                    AccessValidatorUserServiceURL = USER_SERVICE_TEST_REQUEST_URL,
+                    AccessValidatorCheckRightsServiceURL = CHECK_RIGHTS_SERVICE_TEST_REQUEST_URL
+                });
+
+            // allows to pass harness.Bus into the AccessValidator constructor. Otherwise it is null.
```

Comment:
Allows to pass harness. Bus into the AccessValidator constructor. Otherwise it is null.

Reply:
Um.... what? .Bus is the property. Stop nit-picking my PR.

Reply:
Ah, capitalize the first letter. Well that's some big deal problems here alright

Reply:
Fixed the message.